### PR TITLE
ANC Changes - Fundal Height

### PR DESCRIPTION
**Pull request by anvmn**
#626

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-02-08 12-15

**Review by AronNovak — APPROVED** [review]

File: client/src/elm/Backend/Measurement/Model.elm [hunk 1]
```diff
@@ -693,7 +693,8 @@ type alias MedicationValue =
 
 
 type alias ObstetricalExamValue =
-    { fundalHeight : HeightInCm
+    { fundalPulpable : Bool
```
Comment (AronNovak) [inline client/src/elm/Backend/Measurement/Model.elm]:
```suggestion
    { fundalPalpable : Bool
```

File: client/src/elm/Pages/Prenatal/Activity/Model.elm [hunk 2]
```diff
@@ -111,6 +111,7 @@ type Msg
     | SetObstetricalExamFloatMeasurement (Maybe Float -> ObstetricalExamForm -> ObstetricalExamForm) String
     | SetObstetricalExamFetalPresentation FetalPresentation
     | SetObstetricalExamCSectionScar CSectionScar
+    | HideFundalPulpablePopup
```
Comment (AronNovak) [inline client/src/elm/Pages/Prenatal/Activity/Model.elm]:
```suggestion
    | HideFundalPalpablePopup
```

File: client/src/elm/Pages/Prenatal/Activity/Model.elm [hunk 3]
```diff
@@ -883,25 +884,29 @@ emptyNutritionAssessmentForm =
 
 
 type alias ObstetricalExamForm =
-    { fundalHeight : Maybe Float
+    { fundalPulpable : Maybe Bool
```
Comment (AronNovak) [inline client/src/elm/Pages/Prenatal/Activity/Model.elm]:
```suggestion
    { fundalPalpable : Maybe Bool
```

File: client/src/elm/Pages/Prenatal/Activity/Model.elm [hunk 4]
```diff
@@ -883,25 +884,29 @@ emptyNutritionAssessmentForm =
...
+    , fundalHeight : Maybe Float
     , fundalHeightDirty : Bool
     , fetalPresentation : Maybe FetalPresentation
     , fetalMovement : Maybe Bool
     , fetalHeartRate : Maybe Int
     , fetalHeartRateDirty : Bool
     , cSectionScar : Maybe CSectionScar
+    , displayFundalPulpablePopup : Bool
```
Comment (AronNovak) [inline client/src/elm/Pages/Prenatal/Activity/Model.elm]:
```suggestion
    , displayFundalPalpablePopup : Bool
```

Comment (AronNovak) [reply client/src/elm/Pages/Prenatal/Activity/Model.elm]: and so on

**Review by AronNovak — APPROVED** [review]

**Pull request merged**